app/test/testmerginapi.cpp

```diff
@@ -380,6 +438,22 @@ void TestMerginApi::copyTestProject()
   qDebug() << "TestMerginApi::copyTestProject DONE";
 }
 
+void TestMerginApi::verifyDiff( const ProjectDiff &diff, const QString &key )
```

Comment:
what should this do actually? :) check whether "added" or "removed" or "updated" is not empty? I would suggest to have a simpler test like QVERIFY( diff.updated.contains( file_expected_to_be_modified ) );